templates/Settings.tpl

```diff
@@ -0,0 +1,47 @@
+<script type="text/javascript">
+  {literal}
+  (function($) {
+
+    $(function() {
+      moveFields();
+    });
+
+    /**
+     * Re-organise fields
+     */
+    function moveFields() {
+      $('.crm-preferences-display-form-activity_assignee_group_restriction').insertAfter($('.crm-preferences-display-form-activity_types').has('td.description'));
+      $('#group-email-restriction-field-description').insertAfter($('.crm-preferences-display-form-activity_assignee_group_restriction'));
+      $('.crm-preferences-display-form-activity_assignee_cms_user_restriction').insertBefore($('.crm-preferences-display-form-block-activity_assignee_notification_ics'));
+      $('#cms-user-email-restriction-field-description').insertAfter($('.crm-preferences-display-form-activity_assignee_cms_user_restriction'));
+    }
+
+  })(CRM.$);
+  {/literal}
+</script>
+<table>
+  <tr class="crm-preferences-display-form-activity_assignee_group_restriction">
+    <td class="label">
+      <label for="restrict_email_recipient_groups">Restrict email recipients to groups</label>
```

Comment:
@ayush-compuco  Can you add localisation tag to all text? ` {ts}` ... `{/ts}`

Reply:
Updated the template.